test/data/case14/README.md

```diff
@@ -0,0 +1,7 @@
+# Case 14
+
+The destination folder is not owned by the process running the `Copy` operation, but is writeable by that process.
+
+In this case, the Copy operation requires the `NoTemporaryPermChanges` option to be set, otherwise a `chmod` operation is attempted on the destination dir, which will fail.
+
```

Comment:
This text needs updating to say `PermissionControl: DoNothing` rather than `NoTemporaryPermChanges`